include/freerdp/channels/audin.h

```diff
@@ -21,9 +22,116 @@
 #define FREERDP_CHANNEL_AUDIN_H
 
 #include <freerdp/api.h>
+#include <freerdp/codec/audio.h>
 #include <freerdp/dvc.h>
 #include <freerdp/types.h>
 
 #define AUDIN_DVC_CHANNEL_NAME "AUDIO_INPUT"
 
+typedef enum
```

Comment:
why expose this internal detail?
better keep types and structs not necessary for external use away from public headers

Reply:
They are actually part of the spec. I moved those now though to `channels/audin/server/audin.c` and `channels/audin/client/audin_main.c`.